### ISSUE TITLE
CI: add operator tests job for peer-pods

### ISSUE THREAD

**Issue by wainersm** [confidential-containers/operator#336]
**Scenario 1**: Run operator tests for peer-pods on an Ubuntu VM runner
**Scenario 2**: Run operator tests dailyfor peer-pods on an Ubuntu runner

**Use case**: As a developer that opens a pull request to this repository, I want to have the operator installed for `config/samples/ccruntime/peer-pods/` and operator tests ran so that I can be assured my changes don't break the peer-pods installation (**)

We might want to run a peer-pods simple functional (a.k.a smoke) test as well but I believe this can be tracked in another issue. The scope for this one is only to run the operator tests.

Cc @stevenhorsman

**Comment by stevenhorsman** [comment]
Hey @wainersm - thanks for raising this. What are your expectations for this work - to create a new job that installed the peer-pod version of cc runtime and check that the kata-remote runtimeClass is available, or add a new test to the existing jobs to do the same thing?

**Comment by stevenhorsman** [comment]
> I opened this issue thinking on the former: create a new job to run the current operator tests (install, uninstall) with config/samples/ccruntime/peer-pods/ configuration. Later we could use that same job to run some smoke test (e.g. run the simple pod test with libvirt provider) if we think it is a good idea.

> Makes sense?

Yep - that's what I expected, but just wanted to double-check. Thanks!

**Comment by stevenhorsman** [comment]
@wainersm - FYI I knocked together https://github.com/confidential-containers/operator/compare/main...stevenhorsman:peerpods-tests?expand=1 whilst in a meeting. I'm pretty sure that it's not what you are after, but I thought I'd share to get some feedback
